rfc/002-extension-content-configuration-processing.md

```diff
@@ -36,6 +38,8 @@ metadata:
   name: example-extension
   namespace: default
   generation: 1
+  labels:
+    extensionClassName: extension-class-sample1
```

Comment:
Is it enough that we are only defining a name here or should the label also include namespace?

Reply:
I think the label should also include some domain prefix, to avoid clashes.

We normally use `core.openmfp.io` or even more specific `extension.core.openmfp.io`

Reply:
domain and namespace labels are added

Reply:
This is not what i meant @akafazov 

The label should rather look like `extensionclass.openmfp.io: extension-class-sample1`

Reply:
OK changed.